services/admin/assets/js/admin.js

```diff
@@ -33,11 +33,13 @@ var connectionString = function() {
 var getSeriesFromJSON = function(data) {
     var results = [];
     data.results.forEach(function(result) {
-        result.series.forEach(function(s) {
-            results.push(s);
-        });
+        if (result.series) {
+            result.series.forEach(function(s) {
+                results.push(s);
+            });
+        }
     });
-    return results;
+    return results.length > 0 ? results : null;
```

Comment:
It looks like we're actually checking for `null` when this function is called, so this is the right fix for now. It seems that since we're initializing it to `[]` at the top, we should probably change that usage, but that can wait for later.